docker-compose.yml

```diff
@@ -388,25 +412,26 @@ services:
       retries: 5
   oathkeeper:
     image: oryd/oathkeeper:v0.40.4-distroless
+    ports: []
+    extra_hosts: []
     command: serve -c /home/ory/oathkeeper.yml --sqa-opt-out
     volumes:
-      - type: bind
```

Comment:
type bind is by default?

Reply:
yes the -`<origin>:<mountpoint>` is a simpler way to express the same thing.